src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -451,6 +451,11 @@ case class IndexLogEntry(
     relations.head.data.properties.content.fileInfos
   }
 
+  @JsonIgnore
+  lazy val sourceFilesBytes: Long = {
+    sourceFileInfoSet.foldLeft(0L)(_ + _.size)
```

Comment:
nit: you can also do `sourceFileInfoSet.map(_.size).sum` similar to `inputSourceFiles.map(_.size).sum` below. But no strong opinion here (you don't need to change if you don't want).